### PR TITLE
Lava Land terrain generation improvements

### PR DESCRIPTION
**Pull request by PAndaContron**
These are the changes I have made since the original one:

- New custom blocks ash and volcanic rock replace the Core blocks in the previous one

- Naturally generating petrified trees

- Naturally generating house structure made from petrified logs and volcanic rock

- Deadbushes from Core naturally generate

### CODE REVIEW

**Comment by mjuvekar7** [comment]
Hey I can't see any house structures. Also the bad import statement is back

File: src/main/java/org/terasology/lavalandgenerator/WorldGenerator.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package org.terasology.lavalandgenerator;
+import build.classes.org.terasology.core.world.generator.facetProviders.SeaLevelProvider;
```
Comment (mjuvekar7) [inline src/main/java/org/terasology/lavalandgenerator/WorldGenerator.java]: Change this

**Review by mjuvekar7 — CHANGES_REQUESTED** [review]

**Comment by mjuvekar7** [comment]
Also please update your fork. We've added a screenshot directory where screenshots should be stored. Also that will prevent future merge issues

**Comment by PAndaContron** [comment]
I've made the necessary changes to the code, but it turns out one of the screenshots was actually from an older version, so I'll have to wait until I have access to the computer where the screenshots are stored locally before I can replace that.

**Comment by mjuvekar7** [comment]
Looks good! When you get time, you might want to add torches inside the houses. Gets a bit hard to see inside. Might also want to look into using [StructureTemplates](https://github.com/Terasology/StructureTemplates)

**Comment by mjuvekar7** [comment]
Also you should update your fork before working on changes. That should prevent ugly commit trees like the above :)

**Pull request merged**